projects/fusion-ui/src/lib/components/ui-components/dropdown/dropdown/dropdown.component.ts

```diff
@@ -457,7 +458,11 @@ export class DropdownComponent extends StyleBase implements OnInit, OnDestroy, O
                 (!this.autoComplete && !this.backendPaginationState && this.searchValue && this.searchValue.value) ||
                 this.isPredefinedTags
             ) {
-                const filterByArray: string[] = this.mappingOptions ? Object.values(this.mappingOptions) : [];
+                const filterByArray: string[] = this.searchByProperties?.length
```

Comment:
this is not readable. please change to if-else conditions

Reply:
Ok.